src/arch/intel.c

```diff
@@ -57,8 +40,9 @@ static void do_cpuid(unsigned int *eax, unsigned int *ebx, unsigned int *ecx,
 int ceph_arch_intel_probe(void)
 {
 	/* i know how to check this on x86_64... */
-	unsigned int eax = 1, ebx, ecx, edx;
-	do_cpuid(&eax, &ebx, &ecx, &edx);
+	unsigned int eax, ebx, ecx=0, edx=0;
```

Comment:
@tchaikov Fixed the review comments. The reason is only ecx and edx are used below and they shows warnings "Not initialized". But I'm open to your suggestions that will improve the code.